src/pages-sections/why-abu-dhabi/Section1.jsx

```diff
@@ -0,0 +1,27 @@
+import { Box } from "@mui/material";
+import { H1 } from "components/Typography";
+
+const Section1 = () => {
+    return (
+        <Box sx={{
+            background: "url(/assets/images/why-abu-dhabi/Abu-Dhabi-skyline.jpg)",
+            backgroundRepeat: "no-repeat",
+            backgroundSize: "100% 100%",
```

Comment:
this will not scale well in small screen the image will git distorted.
maybe you can try object-fit:cover or something